src/index.js

```diff
@@ -1,27 +1,46 @@
-import * as THREE from 'three';
+import * as THREE from "three";
 
-const scene = new THREE.Scene();
-const camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 0.1, 1000 );
+import Tent from "./Tent.js";
 
-const canvas = document.querySelector('#canvas');
-const renderer = new THREE.WebGLRenderer( { canvas } );
-renderer.setSize( window.innerWidth, window.innerHeight );
-document.body.appendChild( renderer.domElement );
+const scene = new THREE.Scene();
+const camera = new THREE.PerspectiveCamera(
+  75,
+  window.innerWidth / window.innerHeight,
+  0.1,
+  1000
+);
+
+const canvas = document.querySelector("#canvas");
+const renderer = new THREE.WebGLRenderer({ canvas });
+renderer.setSize(window.innerWidth, window.innerHeight);
+document.body.appendChild(renderer.domElement);
 
 const geometry = new THREE.BoxGeometry();
-const material = new THREE.MeshBasicMaterial( { color: 0x00ff00 } );
-const cube = new THREE.Mesh( geometry, material );
-scene.add( cube );
+const material = new THREE.MeshBasicMaterial({ color: 0x00ff00 });
+const cube = new THREE.Mesh(geometry, material);
+
+// adding tent with entrance
+const { group } = new Tent(0, 0, 0);
+scene.add(group);
+
+// test for showing random tents
+for (let i = 0; i < 10; i++) {
+  const randomPos = Math.random() < 0.5 ? -1 : 1;
+  const randomX = Math.floor(Math.random() * randomPos * 10) * i;
+  const randomZ = Math.floor(Math.random() * randomPos * 5) * i;
+  const { group } = new Tent(randomX, 0, randomZ);
```

Comment:
Good! Many people make the mistake to believe `z` is the `up/down` axis but by default it is `y` in Threejs.